liquid/conf/views.py

```diff
@@ -6,14 +6,14 @@
 from django.forms import ValidationError
 from conf.models import Company
 from utils.django_mailman.models import List
-from utils.group_decorator import group_admin_required
+from utils.group_decorator import group_admin_required, check_group_admin
 import simplejson as json
 
 def main(request):
 	return render_to_response('conf/main.html',{"section":"conf", "page":"main"},context_instance=RequestContext(request))
 
 def jobfair(request):
-    if request.user.is_authenticated():
+    if request.user.is_authenticated() and check_group_admin(['Conference', '!Company'], request):
```

Comment:
This seems more like a helper function rather than something that should be checked like this. I would stick to using the decorator to check group membership for permissions.

Also, why do we need this check here? What happens if someone who is not company or corporate accesses this page while logged in? Won't they be redirected to login instead of getting the permission denied?